apps/api/src/company/dto/create-company.dto.ts

```diff
@@ -12,25 +13,39 @@ export class CreateCompanyDto {
 
   @Expose()
   @IsString()
-  @ApiProperty()
+  @ApiProperty({
+    description:
+      'BULSTAT Unified Identification Code (UIC) https://psc.egov.bg/en/psc-starting-a-business-bulstat',
+  })
+  @IsUUID()
```

Comment:
I'm not sure that the `companyNumber` is `UUID` format, maybe a typo

Reply:
Nope, it is not. It's unique in the DB, but not in UUID format.